gc/verbose/schema.xsd

```diff
@@ -624,6 +626,13 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 		<attribute name="timestamp" type="dateTime" use="optional" />
 	</complexType>
 
+	<complexType name="heap-fixup">
+		<attribute name="id" type="integer" use="optional" />
```

Comment:
We don't report id. This is probably a copy-paste from heap-resize which has 2 flavors, and one of them reports ID, hence it's optional here is schema.

ID makes sense to exist if there could be more then one of these elements per cycle. For example, for allocate-stats we don't have it. Very unlikely we could have multiple of them, if possible at all, so I would not bother adding the id in the element, but rather just remove it from the schema.

Reply:
Ok, yes I can do that.